### ISSUE TITLE
[Fargate] [request]: Can we use Fluentd Log driver in fargate without using firelens

### ISSUE THREAD

**Issue by chungath** [aws/containers-roadmap#627]
### Community Note

* Please vote on this issue by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original issue to help the community and maintainers prioritize this request
* Please do not leave "+1" or "me too" comments, they generate extra noise for issue followers and do not help prioritize the request
* If you are interested in working on this issue or have submitted a pull request, please leave a comment

**Tell us about your request**
This is related to the fluentd support in fargate. The earlier issue https://github.com/aws/containers-roadmap/issues/383, got closed pointing to firelens. 

In our case, the ECS containers forward the logs to a fluentd aggregator which sends the logs into a kinesis stream.  

From what I understand firelens does not support kinesis streams yet. So, is there a way to use fluentd directly from fargate containers without using firelens? 

Basically, similar to how it works currently for ECS containers.

**Which service(s) is this request for?**
Fargate

**Additional context**
https://github.com/aws/containers-roadmap/issues/383

**Comment by PettitWesley** [comment]
@chungath FireLens does support Kinesis Streams!

FireLens allows you to use Fluentd or Fluent Bit as your log router. Fluent Bit is our recommendation though. The AWS for Fluent Bit image recently added support for Kinesis Streams: https://github.com/aws/aws-for-fluent-bit/releases/tag/v2.0.0

Furthermore, you can forward to your Fluentd aggregator using Fluent Bit and FireLens: https://github.com/aws-samples/amazon-ecs-firelens-examples/tree/master/examples/fluent-bit/forward-to-aggregator

**Comment by krabradosty** [comment]
@PettitWesley But why just don't implement a support of docker fluentd driver in Fargate? Your solution is just a workaround for me.

**Comment by thisismana** [comment]
As stated [here](https://docs.aws.amazon.com/AmazonECS/latest/developerguide/AWS_Fargate.html#fargate-tasks-logging):
> Amazon ECS task definitions for Fargate support the awslogs, splunk, firelens, and fluentd log drivers for the log configuration. 

But this is not true. I'd also like to just fluent-bit on its own, since it has all the features we need.

**Comment by TestGiri** [comment]
AWS should consider this rather than forcing us to use FireLens
